packages/nibijs/src/testutil.ts

```diff
@@ -22,7 +23,7 @@ export const ERR = {
 }
 
 /** Validates that block queried via the JSON RPC client has the expected fields. */
-export function validateBlockFromJsonRpc(blockJson: any) {
+export function assertValidBlockFromJsonRpc(blockJson: any) {
```

Comment:
testutil renames are for consistency. Some places, we use "validate", others "valid" or "assert". Using "assert" would be consistent with `cosmjs`